2020-02-20 08:21:55 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 08:22:00 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 08:27:11 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 08:27:11 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 08:28:01 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 08:31:33 jdbc[6]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 08:31:37 jdbc[7]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 08:50:50 jdbc[8]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 19:20:34 database: wrong user or password; user: ""
org.h2.message.DbException: Wrong user name or password [28000-199]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:126)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:270)
	at com.github.hcsp.Main.loadUrlsFromDatabase(Main.java:55)
	at com.github.hcsp.Main.main(Main.java:24)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:459)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 16 more
2020-02-20 19:21:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-199]
2020-02-20 11:21:44 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 11:21:45 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 11:21:47 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 19:22:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "0" for parameter "columnIndex" [90008-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:587)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3231)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3270)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:296)
	at com.github.hcsp.Main.loadUrlsFromDatabase(Main.java:61)
	at com.github.hcsp.Main.main(Main.java:26)
2020-02-20 19:23:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-199]
2020-02-20 19:23:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-199]
2020-02-20 19:24:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "0" for parameter "columnIndex" [90008-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:587)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3231)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3270)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:296)
	at com.github.hcsp.Main.loadUrlsFromDatabase(Main.java:61)
	at com.github.hcsp.Main.main(Main.java:26)
2020-02-20 19:26:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-199]
2020-02-20 11:27:26 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 19:27:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-199]
2020-02-20 19:28:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-199]
2020-02-20 19:29:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-199]
2020-02-20 19:29:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-199]
2020-02-20 11:30:45 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 11:31:53 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 11:31:53 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 11:31:54 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 12:40:32 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 20:47:42 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.github.hcsp.Main.main(Main.java:25)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxxstf2934446... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-19%2Fdetail-iimxyqvz4290061.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5352278&cid=10285663&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5386504&cid=10323948&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5402826&cid=10360550&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1036&title=%E9%83%BD%E5%B8%82%E6%A0%A1%E5%9B%AD&pos=108&vt=4' (124)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1003&title=%E7%8E%84%E5%B9%BB%E5%A5%87%E5%B9%BB&pos=108&vt=4' (124)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1006004&title=%E6%80%BB%E8%A3%81%E8%B1%AA%E9%97%A8&pos=108&vt=4... (127)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1006005&title=%E7%A9%BF%E8%B6%8A%E9%87%8D%E7%94%9F&pos=108&vt=4... (127)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5350918&cid=10282637&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5389823&cid=10352585&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5407922&cid=10374366&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5425775&cid=10420598&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5420584&cid=10383898&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5398459&cid=10368966&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxxstf3086653... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxxstf3089358... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxyqvz4528396.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxxstf3089451.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxyqvz4526627.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxyqvz4529554... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchanjing%2Fgsxw%2F2020-02-20%2Fdetail-iimxyqvz4510... (150)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5352278&cid=10285663&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5386504&cid=10323948&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5402826&cid=10360550&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1036&title=%E9%83%BD%E5%B8%82%E6%A0%A1%E5%9B%AD&pos=108&vt=4' (124)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1003&title=%E7%8E%84%E5%B9%BB%E5%A5%87%E5%B9%BB&pos=108&vt=4' (124)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1006004&title=%E6%80%BB%E8%A3%81%E8%B1%AA%E9%97%A8&pos=108&vt=4... (127)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1006005&title=%E7%A9%BF%E8%B6%8A%E9%87%8D%E7%94%9F&pos=108&vt=4... (127)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5427364&cid=10388576&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5390973&cid=10347890&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5420584&cid=10383898&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5398459&cid=10368966&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5391598&cid=10368287&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5402008&cid=10381946&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxxstf3086653... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxxstf3089358... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxyqvz4528396.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxxstf3089451.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxyqvz4526627.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxyqvz4529554... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchanjing%2Fgsxw%2F2020-02-20%2Fdetail-iimxyqvz4510... (150)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5352278&cid=10285663&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5386504&cid=10323948&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5402826&cid=10360550&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1036&title=%E9%83%BD%E5%B8%82%E6%A0%A1%E5%9B%AD&pos=108&vt=4' (124)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1003&title=%E7%8E%84%E5%B9%BB%E5%A5%87%E5%B9%BB&pos=108&vt=4' (124)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1006004&title=%E6%80%BB%E8%A3%81%E8%B1%AA%E9%97%A8&pos=108&vt=4... (127)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1006005&title=%E7%A9%BF%E8%B6%8A%E9%87%8D%E7%94%9F&pos=108&vt=4... (127)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5427364&cid=10388576&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5390973&cid=10347890&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5420584&cid=10383898&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5398459&cid=10368966&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5391598&cid=10368287&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5402008&cid=10381946&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxxstf3086653... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxxstf3089358... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxyqvz4528396.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxxstf3089451.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxyqvz4526627.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxyqvz4529554... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchanjing%2Fgsxw%2F2020-02-20%2Fdetail-iimxyqvz4510... (150)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5352278&cid=10285663&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5386504&cid=10323948&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5402826&cid=10360550&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1036&title=%E9%83%BD%E5%B8%82%E6%A0%A1%E5%9B%AD&pos=108&vt=4' (124)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1003&title=%E7%8E%84%E5%B9%BB%E5%A5%87%E5%B9%BB&pos=108&vt=4' (124)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1006004&title=%E6%80%BB%E8%A3%81%E8%B1%AA%E9%97%A8&pos=108&vt=4... (127)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1006005&title=%E7%A9%BF%E8%B6%8A%E9%87%8D%E7%94%9F&pos=108&vt=4... (127)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5350918&cid=10282637&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5390973&cid=10347890&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5418512&cid=10388695&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5413877&cid=10393535&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5408051&cid=10389923&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5406907&cid=10386420&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxxstf3086653... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxxstf3089358... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxyqvz4528396.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxxstf3089451.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-02-20%2Fdetail-iimxyqvz4526627.d.html&vt=4&po... (132)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchina%2Fgncj%2F2020-02-20%2Fdetail-iimxyqvz4529554... (147)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'https://cj.sina.cn/article/normal_detail?url=https%3A%2F%2Ffinance.sina.cn%2Fchanjing%2Fgsxw%2F2020-02-20%2Fdetail-iimxyqvz4510... (150)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5352278&cid=10285663&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5386504&cid=10323948&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5402826&cid=10360550&pos=108&vt=4&format=simple&nwm=book_wap_0006&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1036&title=%E9%83%BD%E5%B8%82%E6%A0%A1%E5%9B%AD&pos=108&vt=4' (124)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1003&title=%E7%8E%84%E5%B9%BB%E5%A5%87%E5%B9%BB&pos=108&vt=4' (124)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1006004&title=%E6%80%BB%E8%A3%81%E8%B1%AA%E9%97%A8&pos=108&vt=4... (127)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/cate_child.php?cate_id=1006005&title=%E7%A9%BF%E8%B6%8A%E9%87%8D%E7%94%9F&pos=108&vt=4... (127)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5427364&cid=10388576&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5390973&cid=10347890&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5420584&cid=10383898&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5398459&cid=10368966&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5391598&cid=10368287&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 20:48:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100) SELECTIVITY 28": "'http://book.sina.cn/dpool/newbook/bookv1/vipc.php?bid=5402008&cid=10381946&pos=108&vt=4&format=simple&nwm=book_wap_0009&data_id... (134)"; SQL statement:
INSERT INTO NEWS.PUBLIC.LINKS_TO_BE_PROCESSED (link) values(?) [22001-199]
2020-02-20 12:48:26 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 12:48:45 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 12:48:46 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 12:48:57 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 12:52:27 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 12:52:40 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 12:52:40 jdbc[6]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 12:52:48 jdbc[7]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 20:52:57 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.github.hcsp.Main.main(Main.java:25)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-02-20 12:53:35 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 12:53:53 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 12:53:55 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 12:57:40 jdbc[6]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 20:59:24 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.github.hcsp.Main.main(Main.java:25)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-02-20 12:59:43 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 13:00:56 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 13:01:12 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 13:01:23 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 13:01:24 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 13:01:39 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 13:04:25 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 13:04:26 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 13:04:27 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 21:07:31 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.github.hcsp.Main.main(Main.java:25)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-02-20 13:08:19 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 13:08:20 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 13:10:00 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 21:10:28 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.github.hcsp.Main.main(Main.java:25)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-02-20 13:19:45 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-21 00:43:39 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.github.hcsp.Main.main(Main.java:25)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Leo/WorkSpace/hcsp/30_Chapter_Crawler/news.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-02-20 16:46:53 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 16:46:54 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 16:47:08 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2020-02-20 16:47:23 jdbc[6]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1749)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:495)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
